package.json

```diff
@@ -32,10 +32,7 @@
     "fix-bpk-dependencies": "",
     "test": "npm run lint && npm run check-bpk-dependencies && jest --coverage && npm run flow-check-ios && npm run flow-check-android && npm run spellcheck",
     "check-owners": "npm whoami && ( node scripts/npm/get-owners.js | grep -E $(npm whoami) ) && node scripts/npm/check-owners.js",
-    "check-maven-auth": "(cd android && ./gradlew :react-native-bpk-component-calendar:checkMavenCredentials)",
-    "publish": "npm run check-maven-auth && npm run check-owners && git pull --rebase && npm run flow stop && npm test && npm run publish:prepare && lerna publish && npm run publish:bridges",
-    "publish:prepare": "node scripts/npm/native-bridges.js prepare",
-    "publish:bridges": "node scripts/npm/native-bridges.js publish",
+    "publish": "npm run check-owners && git pull --rebase && npm run flow stop && npm test && node scripts/release.js",
```

Comment:
A couple of notes on this from the iOS repository which we have learned over a bunch of failed releases:

+ Check that the branch is [master](https://github.com/Skyscanner/backpack-ios/blob/master/Rakefile#L122)
+ Check [publish rights](https://github.com/Skyscanner/backpack-ios/blob/master/Rakefile#L123), I guess `check-owners` already does that
+ Ensure git branch is [up to date with remote](https://github.com/Skyscanner/backpack-ios/blob/master/Rakefile#L124) via [`branc_up_to_date`](https://github.com/Skyscanner/backpack-ios/blob/458e5527b5ee39d790205c002e51b93876496820/Rakefile#L42-L45)
+ Run a [git fetch](https://github.com/Skyscanner/backpack-ios/blob/458e5527b5ee39d790205c002e51b93876496820/Rakefile#L83) before starting the release process. Doing a pull will not update tags and at least on iOS it's important that you have the latest tagged release available
+ Ensure the repo is [pristine](https://github.com/Skyscanner/backpack-ios/blob/458e5527b5ee39d790205c002e51b93876496820/Rakefile#L128) after running any build steps.

Reply:
Good points! About making sure it's pristine `release-it` does it for us via `requireCleanWorkingDir: true`, I'll look into adding the others

Reply:
I've added all the checks, the branch check is done by the lib as well.